integreat_cms/cms/templatetags/pdf_filters.py

```diff
@@ -30,3 +31,30 @@ def pdf_strip_fontstyles(instance):
         if style := element.attrib.pop("style", None):
             element.attrib["style"] = re.sub(r"font-[a-zA-Z]+:[^;]+", "", style)
     return unescape(tostring(content, with_tail=False).decode("utf-8"))
+
+
+@register.filter
+# pylint: disable=unused-variable
+def pdf_truncate_links(instance, max_chars):
+    """
+    This tag returns the instance with truncated links.
+
+    :param instance: The content object instance
+    :type instance: ~integreat_cms.cms.models.pages.page.Page
+
+    :param max_chars: The maximal length of a link before it will be truncated
+    :type max_chars: int
+
+    :return: The instance with truncated links
+    :rtype: ~integreat_cms.cms.models.pages.page.Page
+    """
+    try:
+        content = fromstring(instance)
+    except ParserError:
+        return instance
```

Comment:
```suggestion
# pylint: disable=unused-variable
def pdf_truncate_links(page_content, max_chars):
    """
    This tag returns the page content with truncated link texts.

    :param page_content: The content of the page
    :type page_content: str

    :param max_chars: The maximal length of a link before it will be truncated
    :type max_chars: int

    :return: The content with truncated links
    :rtype: str
    """
    try:
        content = fromstring(page_content)
    except ParserError:
        return page_content
```
Ah, I just noticed that the types are incorrect.